### DISCUSSION TITLE
Can add our custom stamping images

### DISCUSSION THREAD

**Discussion by ChandreshBaldha** [mozilla/pdf.js#17693] [Q&A]
We need to add our custom stamping images like Approve, Done so that is possible, and if possible then how we implement it.

**Comment by Al00X** [comment]
I also need to implement something like this. Attaching an image using its blob to the PDF programmatically. Would be appreciated if someone directs me.

**Comment by Vaibhav4621** [comment]
@ChandreshBaldha Did you find any solution for your issue?

**Reply by ChandreshBaldha** [reply]
@Vaibhav4621 No I am not able to find any solution.

**Comment by Al00X** [comment]
@Vaibhav4621 @ChandreshBaldha 
I was able to find a solution for this problem. basically I was using [ngx-extended-printer](https://github.com/stephanrauh/ngx-extended-pdf-viewer) package for Angular.
The author of this package added a function for stamping a custom image on the PDF programmatically.

You can see the source codes here:
https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/main/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.service.ts#L564
